### PR TITLE
Homework08.02

### PR DESCRIPTION
**Pull request by sotickmars**
ДЗ 08.02.2020 Колонки через float

### CODE REVIEW

File: project/homework/assets/styles/body.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+h1{
+	text-align: center;
+	font-size: 48px;
+	margin-top: calc(106px - 0.67em);
+	font-family: 'Raleway', sans-serif;
+	color:#333333;
+}
+
+h1:after{
```
Comment (tpotapchik) [inline project/homework/assets/styles/body.css]: ::after

File: project/homework/index.html [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+	<link rel="stylesheet" href="assets/styles/normalize.css">
+	<link rel="stylesheet" href="assets/styles/fonts.css">
+	<link rel="stylesheet" href="assets/styles/body.css">
 	<meta charset="UTF-8">
```
Comment (tpotapchik) [inline project/homework/index.html]: Better to name as main.css or style.css or common.css

File: project/homework/assets/styles/body.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	content:"";
+	display: block;
+	margin:calc(40px - 0.67em) auto 0;
+	height: 2px;
+	width: 64px;
+	background: #ff8c2c;
+}
+
+h1+.content{
```
Comment (tpotapchik) [inline project/homework/assets/styles/body.css]: It’s not a good way to define this selector

File: project/homework/index.html [hunk 4]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+	<link rel="stylesheet" href="assets/styles/normalize.css">
+	<link rel="stylesheet" href="assets/styles/fonts.css">
```
Comment (tpotapchik) [inline project/homework/index.html:6]: Styles should be placed After charset

File: project/homework/assets/styles/body.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
+h1{
+	text-align: center;
+	font-size: 48px;
+	margin-top: calc(106px - 0.67em);
```
Comment (tpotapchik) [inline project/homework/assets/styles/body.css]: We never use calc() for margin

File: project/homework/assets/styles/body.css [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+	height: 2px;
+	width: 64px;
+	background: #ff8c2c;
+}
+
+h1+.content{
+	margin-top: 32px;
+}
+
+.content{
```
Comment (tpotapchik) [inline project/homework/assets/styles/body.css]: As a rule we add space before {

**Review by tpotapchik — CHANGES_REQUESTED** [review]

File: project/homework/assets/styles/body.css (hunk 3 above)
Comment (sotickmars) [reply project/homework/assets/styles/body.css]: Why?

**Review by tpotapchik — APPROVED** [review]

**Pull request merged**